ZENKO-1377: additional tests for ingestionReader

JianqinWang opened the pull request:
- validate tests through kafka

Wait for https://github.com/scality/backbeat/pull/603 to be merged so this pr can be updated accordingly

Commit:
ZENKO-1377: additional tests for ingestionReader
- validate tests through kafka

bert-e commented:
# Hello jianqinwang,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://eve.devsca.com/github/scality/backbeat/bert-e/doc/user).





*Status report is not available.*

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* 2 peers





Peer approvals *must* include at least 1 approval from the following list:

* @jonathan-gramain

* @rahulreddy

* @vrancurel

* @bennettbuchanan

Commit:
ZENKO-1377: move MetadataMock backend to backbeat repo
- This will allow for easier changes for the backbeat ingestion
  tests, as the mock is not used elsewhere
- E2E tests will be using an existing RING instead of a mock
  metadata backend

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* 2 peers





Peer approvals *must* include at least 1 approval from the following list:

* @jonathan-gramain

* @rahulreddy

* @bennettbuchanan

* @vrancurel

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* 2 peers





Peer approvals *must* include at least 1 approval from the following list:

* @rahulreddy

* @bennettbuchanan

* @vrancurel

* @jonathan-gramain

Commit:
ZENKO-1558: fix ingestionReader crash when empty snapshot listing

rahulreddy commented:
@bert-e bypass_author_approval

bert-e commented:
# In the queue



The changeset has received all authorizations and has been added to the
relevant queue(s). The queue(s) will be merged in the target development
branch(es) as soon as builds have passed.

The changeset will be merged in:


* :heavy_check_mark: `development/8.1`



The following branches will **NOT** be impacted:

* `development/7.4`
* `development/7.5`
* `development/8.0`
* `stabilization/7.4.3`



There is no action required on your side. You will be notified here once
the changeset has been merged. In the unlikely event that the changeset
fails permanently on the queue, a member of the admin team will
contact you to help resolve the matter.

*IMPORTANT*

Please do not attempt to modify this pull request.

* Any commit you add on the source branch will trigger a new cycle after the
  current queue is merged.
* Any commit you add on one of the integration branches will be **lost**.

If you need this pull request to be removed from the queue, please contact a
member of the admin team now.




*The following options are set:* **bypass_author_approval**

bert-e commented:
# 


I have successfully merged the changeset of this pull request
into targetted development branches:


* :heavy_check_mark: `development/8.1`



The following branches have **NOT** changed:

* `development/7.4`
* `development/7.5`
* `development/8.0`
* `stabilization/7.4.3`



Please check the status of the associated issue ZENKO-1377.

Goodbye jianqinwang.
